app/views/polls/_comments.html.erb

```diff
@@ -0,0 +1,31 @@
+<% cache [locale_and_user_status, @current_order, commentable_cache_key(@commentable), @comment_tree.comments, @comment_tree.comment_authors, @commentable.comments_count, @comment_flags] do %>
```

Comment:
Why duplicate `/app/views/polls/questions/_comments.html.erb` content in a new file, instead of trying to reuse a single file on both Questions and Polls?

Reply:
At first, it was done that way in case views needed to be tweaked for this specific feature. Nevertheless, since this seems not to be the case, it's better to use actual questions/comments partial instead of duplicate this one. I'll make the changes.